clang/include/clang/Basic/Attr.td

```diff
@@ -4790,7 +4790,7 @@ def HLSLPackOffset: HLSLAnnotationAttr {
 }
 
 def HLSLSV_DispatchThreadID: HLSLAnnotationAttr {
-  let Spellings = [HLSLAnnotation<"SV_DispatchThreadID">];
+  let Spellings = [HLSLAnnotation<"sv_dispatchthreadid">];
```

Comment:
Can we add a check in the tablegen code to error if the HLSLAnnotation attributes ever have a spelling with a capital letter?

I just don't want us to screw this up accidentally and break some new attribute.